catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/associations/Associated.java

```diff
@@ -268,17 +267,18 @@ private Filter forChildAssociations(Metacard metacard) {
             return Collections.emptyMap();
         }
 
-        QueryResponse query = catalogFramework.query(new QueryRequestImpl(new QueryImpl(filter,
+        ResultIterable resultIterable = new ResultIterable(catalogFramework, new QueryRequestImpl(new QueryImpl(filter,
                 1,
                 0,
                 SortBy.NATURAL_ORDER,
                 false,
                 TimeUnit.SECONDS.toMillis(30)), false));
-
-        return query.getResults()
-                .stream()
-                .map(Result::getMetacard)
-                .collect(Collectors.toMap(Metacard::getId, Function.identity()));
+        Map<String, Metacard> results = new HashMap<>();
```

Comment:
@eric chris is right, the iterable cannot use `stream()`, however, if we are wanting to keep the code clean, I could use the `foreach()` method to assign the map
```resultIterable.forEach(result -> results.put(result.getMetacard().getId(), result));```